source/CCustomOpcodeSystem.cpp

```diff
@@ -863,11 +863,10 @@ namespace CLEO
 		CCustomScript *cs = reinterpret_cast<CCustomScript *>(thread);
 		if (thread->IsMission() || !cs->IsCustom())
 		{
-			LOG_WARNING(0, "Incorrect usage of opcode [0A93] in script %s", ((CCustomScript*)thread)->GetInfoStr().c_str());
-
-			return OR_CONTINUE;

```

Comment:
I would keep it as is. Nobody asked for this change

Reply:
Are we going to fix bugs only if somebody else spot and report them?
Why **terminate_this_custom_script** would be ignored in custom script mission?
Why this opcode just prints warning then proceed the execution (potentially random bytes) of the script? Misuse of **004E** was printing error and stopping the script.

Reply:
0a93 never worked in custom missions and nobody used it here. Its been like this from day 1. No need to change the behavior

Reply:
Ok, added old behavior back.